neuralmonkey/runners/label_runner.py

```diff
@@ -15,10 +15,13 @@ def __init__(self,
                 ) -> None:
         super(LabelRunner, self).__init__(output_series, decoder)
         self._postprocess = postprocess
+        decoder.decoded
```

Comment:
nešlo by to tady požehnat nějak líp? :-) osobně mam rád `log("Decoder's decoded tensor is {}".format(decoder.decoded))`